src/js/UnifileService.js

```diff
@@ -299,6 +299,7 @@ export default class UnifileService {
   }
 
   static isService (file) {
-    return typeof file.isLoggedIn !== 'undefined';
+    console.log('isService', file, file.isLoggedIn);
+    return typeof file.mime === 'application/json';
```

Comment:
the console log is a mistake

do you think this is ok for a comment? 

> this mime type is what the server returns when listing services... and I think it never will be the mime type of any other files

Do you think I should rather add a specific field "isService" to the data returned by the server side?

Reply:
I see a lot of use case when a regular file will have a `application/json` MIME type, so yes, it should already be in the data without duck-typing

Reply:
Ok, so do you think I should re-open this PR?
https://github.com/silexlabs/CloudExplorer2/pull/50

Reply:
> I see a lot of use case when a regular file will have a application/json MIME type, so yes, it should already be in the data without duck-typing

for a static file this would be strange
if you remember we decided that this would be the mime type of services, which make them apear with an icon of service

Reply:
You're thinking too much with Silex, not CE. As a developer, my FTP might be full of JSON config files I saved here.

You wouldn't want that to mess up CE